gdcdictionary/schemas/imaging_file.yaml

```diff
@@ -0,0 +1,84 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "imaging_file"
+title: Imaging File
+type: object
+namespace: https://niaid.bionimbus.org
+category: data_file
+program: '*'
+project: '*'
+description: >
+  Data file containing raw imaging data
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - state
+  - created_datetime
+  - updated_datetime
+
+links:
+  - name: imaging_study
+    backref: imaging_files
+    label: data_for
+    target_type: imaging_file
+    multiplicity: many_to_one
+    required: TRUE
```

Comment:
Let’s use consistent capitalization for Boolean? All lowercase?

Reply:
sure, will do